TidyChat/TidyChatPlugin.cs

```diff
@@ -266,9 +274,10 @@ private void OnChat(XivChatType type, int timestamp, ref SeString sender, ref Se
             }
 
             // Don't bother checking if the rule is not active
```

Comment:
Would appreciate updating this comment to more accurately describe the logic of when skips happen - as it can now happen when rules are active.

Reply:
Something like this?

```suggestion
            // Skip rules that wouldn't change isBlocked away from defaultBlocked
```

Do click "Accept suggestion" if you like this change.